### DISCUSSION TITLE
Extract the pathways information with their respective interactions using SPARQL.

### DISCUSSION THREAD

**Discussion by deliamoreno2295** [wikipathways/wikipathways-help#112] [Q&A] [answered]
Hello all,

I don't know if anyone has any idea how I can move the image path in a data frame using sparkql?

I would appreciate it very much.

**Comment by khanspers** [comment, marked as answer]
Hi @deliamoreno2295 , I believe this is related to your question that was answered in the Google group here: https://groups.google.com/g/wikipathways-discuss/c/o7pPyuqVZHg
